core/FileIntegrity.php

```diff
@@ -332,9 +332,9 @@ protected static function isFileNotInManifestButExpectedAnyway($file)
     protected static function getMessagesFilesMismatch($messages)
     {
         $messagesMismatch = array();
-        $hasMd5file = function_exists('md5_file');
+        $hasHashFile = function_exists('hash_file');
         $files = \Piwik\Manifest::$files;
-        $hasMd5 = function_exists('md5');
+        $hasHash = function_exists('hash');
```

Comment:
I would assume we can safely remove this check. The function is already used in many places without a check.